packages/docs/pages/index.vue

```diff
@@ -1,20 +1,142 @@
 <template>
-  <div class="index">
-    Index page
+  <div class="base-layout">
+    <Header :isSidebarVisible.sync="isSidebarVisible" class="base-layout__header" />
+    <div class="base-layout__main">
+      <Sidebar v-if="isSidebarVisible" />
+      <div
+        class="base-layout__content"
+        :class="{'base-layout__content--expanded': !isSidebarVisible} "
+      >
+        <va-breadcrumbs align="left" separator=">" color="gray" class="base-layout__breadcrumbs">
+          <va-breadcrumbs-item
+            v-for="(crumb, index) in crumbs"
+            :label="crumb.label"
+            :to="crumb.path"
+            :key="index"
+            :style="{color: 'gray'}"
+          ></va-breadcrumbs-item>
+        </va-breadcrumbs>
+        <div>
+          <nuxt-child></nuxt-child>
+        </div>
+      </div>
+    </div>
   </div>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+// @ts-nocheck
+import Header from "../components/header/Header.vue";
+import Sidebar from "../components/sidebar/Sidebar.vue";
+import VaBreadcrumbs from "vuestic-ui/src/components/vuestic-components/va-breadcrumbs/VaBreadcrumbs.vue";
+import VaBreadcrumbsItem from "vuestic-ui/src/components/vuestic-components/va-breadcrumbs/VaBreadcrumbsItem.vue";
+import { Component, Vue, mixins } from "vue-property-decorator";
+import { COLOR_THEMES } from '../../ui/src/services/ColorThemePlugin';
 
-@Component({})
+@Component({
+  components: {
+    VaBreadcrumbsItem,
+    VaBreadcrumbs,
+    Header,
+    Sidebar
+  }
+})
 export default class index extends Vue {
```

Comment:
If this index name is required, we can keep this component as a placeholder, and implement layout elsewhere.
i.e.
```vue
<template>
<real-component>
  <nuxt-child>
</real-component>
</template>
```